tests/h/services/flag_test.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.services import flag
+from h import models
+
+
+@pytest.mark.usefixtures('flags')
+class TestFlagServiceFlagged(object):
+    def test_it_returns_true_when_flag_exists(self, svc, flags):
+        user = flags[-1].user
+        annotation = flags[-1].annotation
+
+        assert svc.flagged(user, annotation) is True
+
+    def test_it_returns_false_when_flag_does_not_exist(self, svc, factories):
+        user = factories.User()
+        annotation = factories.Annotation(userid=user.userid)
+
+        assert svc.flagged(user, annotation) is False
+
+    @pytest.fixture
+    def flags(self, factories):
+        return [factories.Flag() for _ in xrange(3)]
```

Comment:
I fixed this by importing `from h._compat import xrange`, I'm guessing that Github/hound is not updating this comment because the fix is on another line.